doc/examples/nonlinear_from_rest/run_simul.py

```diff
@@ -131,6 +137,8 @@ def main(args):
                 f"MC/{dim}D/SFD/Pr_{args.Prandtl:.2f}/asp_{args.aspect_ratio_y:.3f}"
             )
         params.short_name_type_run = f"asp{args.aspect_ratio_y:.3f}_Ra_s{args.Ra_side:.3e}_Ra_v{args.Ra_vert:.3e}_Pr{args.Prandtl:.2f}"
+        if params.oper.dim == 3:
+            params.short_name_type_run = f"Ay{args.aspect_ratio_y:.3f}_Az{args.aspect_ratio_z:.3f}_Ra_s{args.Ra_side:.3e}_Ra_v{args.Ra_vert:.3e}_Pr{args.Prandtl:.2f}"
```

Comment:
Could you please cut these very long lines? It can be done like this:
```python
            params.short_name_type_run = (
                f"Ay{args.aspect_ratio_y:.3f}_Az{args.aspect_ratio_z:.3f}_"
                f"Ra_s{args.Ra_side:.3e}_Ra_v{args.Ra_vert:.3e}_Pr{args.Prandtl:.2f}"
           )
```
Then, use black to format the code again!

```